.ci/jobs/apm-agent-java-load.yml

```
@@ -0,0 +1,6 @@
+##### LOAD TESTING JOB
+- job:
+    name: apm-agent-java/apm-agent-java-mbp
```

It does exist already -> https://apm-ci.elastic.co/job/apm-agent-java/job/apm-agent-java-mbp/

If it's a MBP I'd say to rename also the file with the suffix `-mbp`, otherwise defaults.yml got the `project-type: multibranch` (https://github.com/elastic/apm-agent-java/pull/1475/files#diff-6937eb1bbdbfdece6f197a89b4cec68402f936b219f4b98866a8f6ad2d8fce85R12)

```suggestion
    name: apm-agent-java/apm-agent-java-load-mbp
```